client/src/test/java/org/apache/celeborn/client/ShuffleClientSuiteJ.java

```diff
@@ -195,7 +195,7 @@ private CelebornConf setupEnv(CompressionCodec codec) throws IOException, Interr
     conf.set("rss.client.compression.codec", codec.name());
```

Comment:
It should be changed to "celeborn.shuffle.compression.codec", would you please change it to `SHUFFLE_COMPRESSION_CODEC.key`?